KTL.js

```diff
@@ -8919,6 +9013,35 @@ function Ktl($, appInfo) {
                 })
 
             },
+
+            /** Stick table Headers
+             * @param {string} viewId - The view id
+             * @param {array} data - The data array
+             * @param {number} numOfRecords - The number of records to display
+             * @param {number} viewHeight - The view height */
+            stickTableHeader: function (viewId, data, numOfRecords, viewHeight) {
```

Comment:
`data` and `numOfRecords` are passed only to reject the call of the function. You could stop the execution a lot sooner if you remove that from this function and do that check in the caller function.  Always, the fewer parameters the better.

Reply:
Yes I see that now thank you.